doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -145,6 +145,11 @@ Bluetooth Mesh
  * Added metadata as optional parameter for models Light Lightness Server, Light HSL Server, Light CTL Temperature Server, Sensor Server, and Time Server.
    To use the metadata, enable the :kconfig:option:`CONFIG_BT_MESH_LARGE_COMP_DATA_SRV` Kconfig option.
 
+* Removed:
+
+ * Removed the ``BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE`` Kconfig option as the old APIs based on the :c:struct:`sensor_value` type are removed.
+   Applications using the old APIs must be updated (see the :ref:`migration guide <migration_2_6>`).
```

Comment:
```suggestion
  * Removed the ``BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE`` Kconfig option as the old APIs based on the :c:struct:`sensor_value` type are removed.
    Applications using the old APIs must be updated (see the :ref:`migration guide <migration_2_6>`).
```

Reply:
Do we need an update for migration guide which has been there since v2.6 ?

Reply:
Ah, you're removing a deprecated API. Fair enough. Edited my suggestion (audio is the closest link I could get, but it still works)

Reply:
Can't we add a tag there in migration guide for mesh?

Reply:
We can, but the migration guide was already released with 2.6.0, and we can't edit that release anymore, so adding a tag to latest would be only a partial fix and create a discrepancy. Using this audio tag will still display Mesh entry pretty much at the center.

Reply:
Leaving as it is then.